scripts/tasks/danger.js

```diff
@@ -34,6 +34,5 @@ spawn(
   } else {
     console.log('Danger passed');
   }
-
```

Comment:
Looking at the code style guideline a blank line after a control block is expected https://github.com/airbnb/javascript#whitespace--after-blocks

Reply:
Okay thanks